protos/dcs/mission/v0/mission.proto

```diff
@@ -355,6 +372,8 @@ message StreamEventsResponse {
     DisconnectReason reason = 2;
   }
 
+  message MissionCommandEvent { string id = 1; }
```

Comment:
The other day, I spotted this discussion in the archives: https://github.com/DCS-gRPC/rust-server/discussions/67#discussioncomment-1587771

Having a "marker" for the `callback` is a useful establishment over the loose `google.protobuf.Struct` for contract reasons.

Suggestions I would offer:
* Rename `id` to `stateId`.
* Rebuild whether it is a global / coalition / group level `stateId`. This can be enclosed in the parameters made in the `addCommand` step.

Reply:
`stateId` sounds wierd and I don't really understand the reasoning for it. I am also not sure what the "marker" is about. Could you add some more clarification around the reasoning for this suggestion?

Reply:
Consider the client code has something like the following pseudo:
```
var commandTable = new Dictionary<string, Action>();
// key is state ID
var enrolledDateTime = (new DateTime()).ToString();
var commandStateId = Guid.New().ToString();
commandTable.Add(commandStateId, () => Console.WriteLine("Hello " + enrolledDateTime));

// ...
grpcDcs.AddMissionCommand({StateId = commandStateId});

// ...
grpcDcs.OnMissionEvent(theEvent => {
    var theCommand = theEvent as MissionCommandEvent;
    if (theCommand != null) {
       Action restoredState = commandTable[theCommand.stateId];
       restoredState();
    }
```
The main idea is that it makes the client application responsible in restoring the state for the event, rather than relying on the prior state being passed around from the gRPC->DCS->gRPC layer. It removes the need to have an open struct as mentioned in the discussion I linked to, thus eliminate the dumping group that a mission command could become.

Reply:
Talked in Discord. My suggestion here is https://github.com/DCS-gRPC/rust-server/pull/98/files?diff=split&w=1#r780785475 in the request and in the CommandEvent we just use `name` or `command_name` instead of `id`

Reply:
I don't like the idea of having to keep track which id corresponds to which logic in my client. So I'd personally either use the field to send JSON encoded data (thus the suggestion to rename it to `command` in another comment), or'd suggest to use something like:

```proto3
string command = 1;
repeated google.protobuf.Struct args = 2;
```

I don't have a strong opinion on whether to keep the single string field, or the suggestion above, both would work for my use-cases equally fine. But I am on-board of using another name for the field.